gtfu/src/main/java/gtfu/tools/UpdateTripNames.java

```diff
@@ -66,6 +66,9 @@ public static void UpdateTripNames(String[] agencyIDList) throws Exception {
                 String utf8 = StandardCharsets.UTF_8.name();
                 try (PrintStream ps = new PrintStream(baos, true, utf8)) {
                     TripListGenerator.generateTripList(agencyID, null, ps, false);
+                } catch (Exception e) {
+                    reporter.addLine("   * exception in generating triplist for " + agencyID);
```

Comment:
Might as well add the exception to the error line, e.g. `  * exception generating triplist for " + agencyID + ": " + e`